### ISSUE TITLE
Creating clusters fails while logging request

### ISSUE THREAD

**Issue by oleschoenburg** [zeebe-io/zeebe-cluster-testbench#605]
From the stacktrace it looks like we are trying to create a new cluster but logging the request payload via `EntityLoggingFilter` fails. 
Might be related to https://github.com/zeebe-io/zeebe-cluster-testbench/commit/f54f1156f8428e9b4418235fda2e577bc20ca888.

```
jakarta.ws.rs.ProcessingException: java.lang.NullPointerException: Cannot invoke "java.io.OutputStream.close()" because the return value of "org.jboss.resteasy.util.DelegatingOutputStream.getDelegate()" is null
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.filterRequest(ClientInvocation.java:780) ~[testbench.jar:3.0.0-SNAPSHOT]
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.invoke(ClientInvocation.java:491) ~[testbench.jar:3.0.0-SNAPSHOT]
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invokeSync(ClientInvoker.java:152) ~[testbench.jar:3.0.0-SNAPSHOT]
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invoke(ClientInvoker.java:115) ~[testbench.jar:3.0.0-SNAPSHOT]
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientProxy.invoke(ClientProxy.java:76) ~[testbench.jar:3.0.0-SNAPSHOT]
	at jdk.proxy2.$Proxy33.createCluster(Unknown Source) ~[?:?]
	at io.zeebe.clustertestbench.handler.CreateClusterInCamundaCloudHandler.handle(CreateClusterInCamundaCloudHandler.java:35) ~[testbench.jar:3.0.0-SNAPSHOT]
	at io.camunda.zeebe.client.impl.worker.JobRunnableFactory.executeJob(JobRunnableFactory.java:44) ~[testbench.jar:3.0.0-SNAPSHOT]
	at io.camunda.zeebe.client.impl.worker.JobRunnableFactory.lambda$create$0(JobRunnableFactory.java:39) ~[testbench.jar:3.0.0-SNAPSHOT]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.io.OutputStream.close()" because the return value of "org.jboss.resteasy.util.DelegatingOutputStream.getDelegate()" is null
	at org.jboss.resteasy.util.DelegatingOutputStream.close(DelegatingOutputStream.java:60) ~[testbench.jar:3.0.0-SNAPSHOT]
	at java.io.FilterOutputStream.close(Unknown Source) ~[?:?]
	at io.zeebe.clustertestbench.cloud.filter.EntityLoggingFilter.filter(EntityLoggingFilter.java:42) ~[testbench.jar:3.0.0-SNAPSHOT]
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.filterRequest(ClientInvocation.java:767) ~[testbench.jar:3.0.0-SNAPSHOT]
	... 14 more
	Suppressed: java.lang.NullPointerException: Cannot invoke "java.io.OutputStream.flush()" because the return value of "org.jboss.resteasy.util.DelegatingOutputStream.getDelegate()" is null
		at org.jboss.resteasy.util.DelegatingOutputStream.flush(DelegatingOutputStream.java:54) ~[testbench.jar:3.0.0-SNAPSHOT]
		at java.io.FilterOutputStream.flush(Unknown Source) ~[?:?]
		at java.io.FilterOutputStream.close(Unknown Source) ~[?:?]
		at io.zeebe.clustertestbench.cloud.filter.EntityLoggingFilter.filter(EntityLoggingFilter.java:42) ~[testbench.jar:3.0.0-SNAPSHOT]
		at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.filterRequest(ClientInvocation.java:767) ~[testbench.jar:3.0.0-SNAPSHOT]
		at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.invoke(ClientInvocation.java:491) ~[testbench.jar:3.0.0-SNAPSHOT]
		at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invokeSync(ClientInvoker.java:152) ~[testbench.jar:3.0.0-SNAPSHOT]
		at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invoke(ClientInvoker.java:115) ~[testbench.jar:3.0.0-SNAPSHOT]
		at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientProxy.invoke(ClientProxy.java:76) ~[testbench.jar:3.0.0-SNAPSHOT]
		at jdk.proxy2.$Proxy33.createCluster(Unknown Source) ~[?:?]
		at io.zeebe.clustertestbench.handler.CreateClusterInCamundaCloudHandler.handle(CreateClusterInCamundaCloudHandler.java:35) ~[testbench.jar:3.0.0-SNAPSHOT]
		at io.camunda.zeebe.client.impl.worker.JobRunnableFactory.executeJob(JobRunnableFactory.java:44) ~[testbench.jar:3.0.0-SNAPSHOT]
		at io.camunda.zeebe.client.impl.worker.JobRunnableFactory.lambda$create$0(JobRunnableFactory.java:39) ~[testbench.jar:3.0.0-SNAPSHOT]
		at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:?]
		at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:?]
		at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) ~[?:?]
		at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
		at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
		at java.lang.Thread.run(Unknown Source) ~[?:?]
```

**Comment by oleschoenburg** [comment]
@pihme I had a quick look at this but I'm not sure what the issue is exactly. Could you take over?

**Comment by pihme** [comment]
I reverted the commit. I thought about catching the NPE or doing something else, but after careful consideration I believe the best course of action is to revert the change.

So now code scanner will complain that
```
      final var stream = new LoggingStream(requestContext.getEntityStream());
      requestContext.setEntityStream(stream);
      requestContext.setProperty(ENTITY_STREAM_PROPERTY, stream);
```
stream is not always closed. I am not sure when the stream is closed, but the most likely scenario in my mind is that the stream is closed during request processing. Closing the stream during filtering clearly doesn't make sense.
